### ISSUE TITLE
tradeogre fees

### ISSUE THREAD

**Issue by hulitolku** [Endogen/TradeOgrePy#3]
How i can get fees from tradeogre api?

**Comment by Endogen** [comment]
You can't. There is no way to retrieve fees from the API directly. You would need to scrape it from [here](https://tradeogre.com/help/fees) which isn't particularly hard to do but not included in TradeOgres API.

**Comment by hulitolku** [comment]
> from [here](https://tradeogre.com/help/fees)

This page loading from cloudflare.
Requests dont work.

**Comment by Endogen** [comment]
It works for me but maybe you are blocked by Cloudflare. Try it with VPN. But again, unfortunately it's not part of the API.
